### ISSUE TITLE
Add categories

### ISSUE THREAD

**Issue by talha131** [Pelican-Elegant/documentation#17]
Currently we have all articles filed under "Elegant - Pelican Theme". Originally documentation was hosted at my personal blog. It made sense to file everything under "Elegant" category.

But this needs to be expanded now. Off the top of my head

1. Configuration 
1. Guides (It will have articles on integrating plugins with the theme)
1. Customization
1. Analytics

Suggestions and PR are welcomed.

**Comment by silverhook** [comment]
This sounds like a good overview, yes.

Are these already pages or blog entries?

In any case, it might make sense to add a blog for releases (and big announcements).

**Comment by talha131** [comment]
We need to change meta data of existing articles.

Announcement articles are good suggestion. We can write release notes articles and file them under "Announcement" category.

**Comment by silverhook** [comment]
Another good category would be “Release notes”. Especially for people who would only want to follow our releases, and nothing else.

I’ll add it for #52, if no-one complains.

**Comment by talha131** [comment]
Thanks @silverhook. Yeah, "Release Notes" category is fine. It's more apt for #52 than "Announcement" category.

**Comment by silverhook** [comment]
Should we re-order the categories as folders, for easier navigation?

Currently the categories are explicitly stated in the MarkDown files, but we could just create the appropriate folders and move them there.

**Comment by talha131** [comment]
Yes. You are right. Each category should get its folder. Currently it isn't b/c its originally from my blog which placed all elegant related material inside elegant folder.

**Comment by silverhook** [comment]
To recap, we have two options:

# All blogs

One option would be to keep everything as articles and have the following categories (separated as folders):

- Configuration (and installation)
- Guides and HowTos (It will have articles on integrating plugins with the theme)
- Customization
- Analytics
- Release Notes
- Announcements

# Blogs and pages

Or have pages and blogs entries separated:

Blog entries for stuff where the publication date is useful info:

- Release notes
- Announcements

Pages, where publication date is not a relevant piece of info, or even counterproductive (e.g. people think that configuration page is outdated):

- Configuration (and installation)
- Guides and HowTos (It will have articles on integrating plugins with the theme)
- Customization
- Analytics
